WordPress/Classes/Networking/AccountServiceRemoteREST.m

```diff
@@ -113,6 +117,72 @@ - (void)updateBlogsVisibility:(NSDictionary *)blogs
            }];
 }
 
+- (WordPressComApi *)anonymousApi
+{
+    if (!_anonymousApi) {
+        _anonymousApi = [WordPressComApi anonymousApi];
+    }
+
+    return _anonymousApi;
+}
+
+- (void)findExistingAccountByEmail:(NSString *)email success:(void (^)(BOOL found))success failure:(void (^)(NSError *error))failure
+{
+    // TODO: We need to make a versioned flavor of this endpoint and ensure it always returns a JSON object.
+    // TODO: Remove the special case in `WordPressComApi.assertApiVersion` the endpoint is versioned.
```

Comment:
We usually sign our `TODO`s with a name/date. Bonus points for a link to Github issue :wink: